<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEqualizerDialog</class>
 <widget class="QDialog" name="CEqualizerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>165</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>343</width>
    <height>165</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>343</width>
    <height>165</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Parametric Equalizer</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Settings</string>
   </property>
   <widget class="QLabel" name="reverbLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>reverb</string>
    </property>
   </widget>
   <widget class="QSlider" name="reverbSlider">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>29</y>
      <width>22</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="l8khzLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>8 khz</string>
    </property>
   </widget>
   <widget class="QSlider" name="s125hzSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>22</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="l1khzLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1 khz</string>
    </property>
   </widget>
   <widget class="QLabel" name="l125hzLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>125 hz</string>
    </property>
   </widget>
   <widget class="QSlider" name="s8khzSlider">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>29</y>
      <width>22</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSlider" name="s1khzSlider">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>29</y>
      <width>22</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="onoffButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>on/off</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l4khzLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4 khz</string>
    </property>
   </widget>
   <widget class="QSlider" name="s4khzSlider">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>29</y>
      <width>22</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="l250hzLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>250 hz</string>
    </property>
   </widget>
   <widget class="QSlider" name="s250hzSlider">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>22</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
